ui/components-react/src/components-react/converters/NumericTypeConverter.ts

```diff
@@ -68,7 +73,7 @@ export class FloatTypeConverter extends NumericTypeConverterBase {
   }
 
   public override convertFromString(value: string): number {
```

Comment:
@bsteinbk @NancyMcCallB I would like to change signature of this method to return `number | undefined` and return `undefined` when value is not a valid number. What do you think about this? Should it return `NaN` and not default to `undefined` or `0`?

Also it looks like it would be a breaking change. However I doubt anyone is using `FloatTypeConverter` or `IntTypeConverter` directly outside `TypeConverterManager` so maybe it could be done?

Reply:
@saskliutas - I think returning undefined is a good change and likely to be encountered in editing scenarios. While Nan would also work, it may be more easily overlooked and not tested for. I believe you are also correct that there are very few current users of type converters in general so now is the best time to make this change. @NancyMcCallB do you agree?

Reply:
Agree with @bsteinbk.

Reply:
Changed signature and method to return `undefined` when `NaN` is encountered